From c51a3fa53e4b13e8adcead066e59c2b464fd697f Mon Sep 17 00:00:00 2001
From: Xiretza <xiretza@xiretza.xyz>
Date: Sun, 23 Jan 2022 13:02:56 +0100
Subject: [PATCH 2/2] Makefile: always use host gprinstall

By default, the Makefile tries to use the freshly-built gprinstall to
install itself - but since that binary required libgpr, which we are
currently building, this can't work.

Simply fall back to gprbuild-bootstrap's statically linked gprinstall.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index d0750332..85b7cca4 100644
--- a/Makefile
+++ b/Makefile
@@ -40,7 +40,7 @@ LIB_DIR       = lib/
 # target options for cross-build
 ifeq ($(HOST),$(TARGET))
 GTARGET=
-INSTALLER=exe/$(BUILD)/$(LIB_INSTALLER)
+INSTALLER=$(LIB_INSTALLER)
 else
 GTARGET=--target=$(TARGET)
 INSTALLER=$(LIB_INSTALLER)
-- 
2.34.1

